### DISCUSSION TITLE
erweiterte Einstellungen - "Redesign"

### DISCUSSION THREAD

**Discussion by jp112sdl** [OpenCCU/OpenCCU#2065] [🇩🇪 Ideen]
Bevor ich einen PR (möglicherweise umsonst) erstelle, frage ich mal die Runde, ob nicht sinnvoller ist, die Hilfetexte über den ?-Button einblenden zu lassen.
Das macht das Fenster viel schlanker und aufgeräumter.

<img width="540" alt="advanced3" src="https://user-images.githubusercontent.com/18190139/203544343-36f34d13-df9f-4966-835d-cffe68d183f4.png">
<img width="540" alt="advanced1" src="https://user-images.githubusercontent.com/18190139/203544346-de41bb0e-f10f-46e8-ab21-ef6c26b65e70.png">
<img width="540" alt="advanced2" src="https://user-images.githubusercontent.com/18190139/203544350-e43e663b-aa7c-49b8-9113-6b2db680db01.png">

Für Devconfig gibt es den Button natürlich auch

**Comment by jens-maus** [comment]
Das würde ich persönlich nicht so chic bzw. intuitiv finden. Heutzutage sollte doch horizontaler Platz auf dem Monitor fast kein Problem mehr sein und so kann man das direkt side-by-side miteinander vergleichen und die Infos zu den vielen Optionen einsehen. Auch verdeckt so ein modales Fenster ja immer gleich die gesamte restliche Funktion der darunterliegenden Fenster und so kan man das nicht parallel offen halten und die optionen schritt-für-schritt durchgehen.

Einzig alleine die Möglichkeit den rechten Info-Bereich vllt. irgendwie auszublenden bzw. einblendbar zu machen halte ich hier für einen gangbaren Weg.

**Reply by jp112sdl** [reply]
> Einzig alleine die Möglichkeit den rechten Info-Bereich vllt. irgendwie auszublenden bzw. einblendbar zu machen halte ich hier für einen gangbaren Weg.

Wäre das ein Kompromiss? Default = Collapsed

![](http://g.recordit.co/Yd0VDYtUAK.gif)

**Reply by jens-maus** [reply]
Das sieht schon besser aus. Einzig alleine das ">" im Button gefällt mir nicht. Entweder nur "Hilfe" oder sowas rein mit dem typischen "?" Icon. Und die Frage wäre noch ob man das auchgepoppt bekommt mit einem "?" Hilfeicon innerhalb des Textes bzw bei System-Options usw wie du es in der ersten Variante hattest. Dann würde ich das vmtl zusätzlich so noch umsetzen.

**Reply by jp112sdl** [reply]
> Und die Frage wäre noch ob man das auchgepoppt bekommt mit einem "?" Hilfeicon innerhalb des Textes 

Hilfe-Icons in der rechten/mittleren Spalte zu platzieren habe ich aufgegeben.
Mal schiebt sich alles nach links, mal nach rechts... da bin ich mit der Style-Formatierung überfordert. An die Style-Klassen selbst wollte ich nicht auch noch rangehen.

> bzw bei System-Options usw wie du es in der ersten Variante hattest. Dann würde ich das vmtl zusätzlich so noch umsetzen.

Also so
![grafik](https://user-images.githubusercontent.com/18190139/203719100-a2e921d7-fff0-4436-9b1d-f92e29dadec3.png)

**Reply by jens-maus** [reply]
Genau, so sieht das sicher besser aus und dann beim drücken auf "?" eben auch einblenden/ausblenden lassen. Und geht das "?" Symbol auch neben dem "Hilfe" Text im Button?

**Reply by jp112sdl** [reply]
> Und geht das "?" Symbol auch neben dem "Hilfe" Text im Button?

So

![grafik](https://user-images.githubusercontent.com/18190139/203721994-ee8f1cfc-e7cb-409b-8481-6a2f26007e0a.png)

oder so:

![grafik](https://user-images.githubusercontent.com/18190139/203722543-de68b623-d7ed-492f-b8f8-14819868bab1.png)


Und noch mal zur Klarstellung: Beim Klicken auf ? unter Watchdog-/System-/Experten-Optionen soll was passieren?
- ein Popup mit der Hilfe zu der Kategorie
- einblenden der gesamten Hilfe wie beim Button im Footer unten rechts

**Reply by jens-maus** [reply]
> oder so:
> 
> ![grafik](https://user-images.githubusercontent.com/18190139/203722543-de68b623-d7ed-492f-b8f8-14819868bab1.png)

Genau so, d.h. "Hilfe (?)". Oder sollten wir das Info nennen?!?! Grübel ich dann beim PR nochmal drüber nach...

> Und noch mal zur Klarstellung: Beim Klicken auf ? unter Watchdog-/System-/Experten-Optionen soll was passieren?
> 
> * ein Popup mit der Hilfe zu der Kategorie
> * einblenden der gesamten Hilfe wie beim Button im Footer unten rechts

letzteres. D.h. da soll genau das gleiche passieren ein/ausblenden der Hilfeteste, kein Popup.
